src/components/TopBar/ItemContainer.js

```diff
@@ -10,7 +10,7 @@ export default styled(Tabbable).attrs({
   px: 3,
   ml: 0,
   alignItems: 'center',
-  cursor: p => (p.disabled ? 'default' : 'pointer'),
+  cursor: 'default',
```

Comment:
remove line (maybe add cursor `not-allowed` if disabled?)
wdyt @khalilbenihoud ?

Reply:
personally not a big fan of not-allowed cursors. especially on windows if i recall correctly. it's kinda overkill as the UI already show it 🤔

Reply:
so if we choose to not use `not-allowed`, we should remove it from side menu item

Reply:
done